std_execution.bs

```diff
@@ -3032,6 +3174,48 @@ are all well-formed.
 
     3. Returns an operation state `op_state` that contains `op_state2`. When `execution::start(op_state)` is called, calls `execution::start(op_state2)`.
 
+#### `execution::done_as_optional` <b>[execution.senders.adaptors.done_as_optional]</b> #### {#spec-execution.senders.adaptors.done_as_optional}
+
+1. `execution::done_as_optional` is used to handle a done signal by mapping it into the value channel as an empty optional. The value channel is also converted into an optional. The result is a sender that never completes with done, reporting cancellation by completing with an empty optional.
+
+2. The name `execution::done_as_optional` denotes a customization point object. For some subexpression `s.`, let `S` be `decltype((s))`. If the type `S` does not satisfy <code><i>single-typed-sender</i></code>, `execution::done_as_optional(s)` is ill-formed. Otherwise, the expression `execution::done_as_optional(s)` is expression-equivalent to:
```

Comment:
You mention that it is a customisation-point-object but do not actually describe the mechanism by which it can be customised. Is it not actually intended to be customisable?

Reply:
This is a quirk of the term. Not all CPOs in the stdlib are customizable. That's just what we call them. Blame me and Casey for that.